$NetBSD$

http_path is not needed (PATH is responsibility of cgi-bin programs)

--- src/nhttpd/http.c.orig	2011-05-31 18:48:32.000000000 +0200
+++ src/nhttpd/http.c	2013-09-15 09:57:07.000000000 +0200
@@ -483,8 +483,6 @@
 			    rh->rq_docrootv);
 			cgienv[i++] = sys_benv("GATEWAY_INTERFACE=%s",
 			    http_fv_cgi);
-			cgienv[i++] = sys_benv("PATH=%s",
-			    http_path);
 			cgienv[i++] = sys_benv("REMOTE_ADDR=%s",
 			    c[sfd].ip);
 			cgienv[i++] = sys_benv("REMOTE_PORT=%s",
